CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ And please only add new entries to the top of this list, right below the `# Unre
 
 # Unreleased
 
+- Added `Occluded(bool)` window event (current only supported on macos) and on macOS, added `WindowExtMacOS::is_occluded`
```

Comment:
```suggestion
- On macOS, added `WindowEvent::Occluded(bool)` and `WindowExtMacOS::is_occluded`.
```
I think this would be more concise, but I'm not sure to what degree we've used this sort of formulation for sparsely-supported platform-independent APIs in the past.

Reply:
I don't know if I agree with this. Saying "On macOS, added `WindowEvent::Occluded(bool)`" in my opinion seems to imply this windows event is only accessible on macOS. Users attempting to match `WindowEvent` will have to handle the `Occluded` event on all platforms (or just `_ =>`) , and other platforms can send this event in the future (ie [wayland](https://gitlab.freedesktop.org/wayland/wayland-protocols/-/merge_requests/99))